Why: Force /wireshark as the data directory (preloaded into the Emscripten FS); needed for preferences, profiles and color filters.

diff --git a/wsutil/filesystem.c b/wsutil/filesystem.c
index f1c7358a97df18a820c4481bd7960a6b1387f4f2..de7ceae8ecfee2ba71fa076cb24737e1abe03de2 100644
--- a/wsutil/filesystem.c
+++ b/wsutil/filesystem.c
@@ -1010,8 +1010,10 @@ get_current_working_dir(void)
 const char *
 get_datafile_dir(void)
 {
-    if (datafile_dir != NULL)
-        return datafile_dir;
+    if (datafile_dir == NULL)
+        datafile_dir = g_strdup(DATA_DIR);
+    
+    return datafile_dir;
 
     char *data_dir_envar = configuration_environment_variable("DATA_DIR");
     if (g_getenv(data_dir_envar) && !started_with_special_privs()) {
